minifi_main/MiNiFiWindowsService.h

```diff
@@ -21,6 +21,8 @@
 
 #include <memory>
 
+using HANDLE = void*;
```

Comment:
could we include some Windows header instead of this?

Reply:
https://github.com/apache/nifi-minifi-cpp/pull/1617#discussion_r1298336482